### PR TITLE
Role graph

### PR DESCRIPTION
**Pull request by kgilpin**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (C) 2013 Conjur Inc
```
Comment (kgilpin) [inline gulpfile.js]: It's 2015 now :-)

File: lib/conjur/authn.js [hunk 2]
```diff
@@ -18,34 +18,36 @@
...
-        g.rest.post(path, { data: apiKey }).on('complete', function(token, response) {
-          if ( token instanceof Error)
-            return callback(token);
-          else if ( response.statusCode !== 200 )
-            return callback(g.format("POST %s failed : %s", path, response.statusCode));
-          
-          callback(null, token);
-        })
-      }    
+    connect: function (url) {
```
Comment (kgilpin) [inline lib/conjur/authn.js]: Oh no, the dreaded global indent reformatting

**Comment by kgilpin** [comment]
In the README can you show how to use the gulp stuff?

**Comment by inscitiv-ops-dev** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.inscitivops.com//job/api-node-pulls/1/
Test FAILed.

**Comment by kgilpin** [comment]
Retest this please

**Comment by inscitiv-ops-dev** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.inscitivops.com//job/api-node-pulls/4/
Test FAILed.

**Comment by inscitiv-ops-dev** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.inscitivops.com//job/api-node-pulls/5/
Test FAILed.

**Comment by inscitiv-ops-dev** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.inscitivops.com//job/api-node-pulls/7/
Test PASSed.

**Comment by inscitiv-ops-dev** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.inscitivops.com//job/api-node-pulls/9/
Test PASSed.

**Pull request merged**